requirements.txt

```diff
@@ -1,12 +0,0 @@
-flake8==3.5.0
-flake8-isort==2.3
-funcsigs==1.0.2
-mozlog==3.7
-PyPOM==1.2.0
-pytest==3.3.2
-pytest-metadata==1.5.1
-pytest-selenium==1.11.4
-pytest-variables==1.7.1
-pytest-xdist==1.22.0
-querystringsafe_base64==1.0.0
-selenium==3.8.1
```

Comment:
If we want to keep pyup support, we need to tell it about the Pipfile by creating a .pyup.yml file:

```yml
requirements:
  - Pipfile
  - Pipfile.lock
```

Reply:
I've got that in a followup PR, but will fold that in here, as it makes sense; thanks!

Reply:
Added in https://github.com/mozilla/stubattribution-tests/pull/106/commits/efadb7ef26c7163d452ff8bc1b30dc72834e6daa#diff-50f285e3023eb0b37f3dc8671b7dace9